pkg/kritis/apis/kritis/v1beta1/attestationauthority.go

```diff
@@ -33,10 +33,10 @@ type AttestationAuthority struct {
 
 // AttestationAuthoritySpec is the spec for a AttestationAuthority resource
 type AttestationAuthoritySpec struct {
-	NoteReference        string `json:"noteReference"`
-	PrivateKeySecretName string `json:"privateKeySecretName"`
-	PublicKeyData        string `json:"publicKeyData"`
-	PolicyType           string `json:"policyType"`
+	NoteReference        string   `json:"noteReference"`
+	PrivateKeySecretName string   `json:"privateKeySecretName"`
+	PublicKeyList        []string `json:"publicKeyList"`
```

Comment:
Do you think we can immediately change this to a key wrapper like [1]? (We'll probably want to add PKIX support soon enough, so might as well prepare for that.)

[1] https://cloud.google.com/binary-authorization/docs/reference/rest/v1/projects.attestors#AttestorPublicKey

Reply:
I think we should probably address this in a separate PR. I documented it in #453 .
WDYT?

Reply:
sgtm